glean-core/csharp/Glean/Metrics/StringListMetricType.cs

```diff
@@ -0,0 +1,172 @@
+﻿using Mozilla.Glean.FFI;
+using System;
+using System.Text.Json;
+
+namespace Mozilla.Glean.Private
```

Comment:
Oh, almost forgot. Please kindly add an entry to [our changelog](https://searchfox.org/glean/source/CHANGELOG.md).